tools/pipelines/test-real-service-stress.yml

```diff
@@ -123,3 +123,45 @@ stages:
         env:
           fluid__test__driver__frs: $(automation-fluid-test-driver-frs-stress-test)
           FLUID_TEST_LOGGER_PKG_PATH: ${{ variables.testWorkspace }}/node_modules/@ff-internal/aria-logger # Contains getTestLogger impl to inject
+
+  # stress tests large payloads odsp
```

Comment:
How do we evaluate this change? It will (if I got it right) cause more runs against ODSP / SPDF, which may result in more throttling. Is it enable and see what happens? Please be prepared to back out :)